lib/validate_params/configuration.rb

```diff
@@ -0,0 +1,17 @@
+module ValidateParams
+  class Configuration
+    attr_accessor :scrub_invalid_utf8, :scrub_invalid_utf8_replacement
+
+    def initialize
+      @scrub_invalid_utf8 = false
+      @scrub_invalid_utf8_replacement = ""
+    end
+
+    def as_json
```

Comment:
Why do we call this method `as_json` when in reality it returns just a hash, would it make more sense to call it `to_h`?